feature/aft/aft_summary/otg_tests/scale_aft_summary/metadata.textproto

```diff
@@ -0,0 +1,54 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid: "89da0b4c-9a16-44f8-9757-d98ccdd6aaf4"
+plan_id: "RT-4.11"
+description: "AFTs Route Summary"
+testbed: TESTBED_DUT_ATE_2LINKS
+platform_exceptions: {
+  platform: {
+    vendor: NOKIA
+  }
+  deviations: {
+    isis_multi_topology_unsupported: true
+    isis_interface_level1_disable_required: true
+    missing_isis_interface_afi_safi_enable: true
+    isis_restart_suppress_unsupported: true
+    explicit_port_speed: true
+    explicit_interface_in_default_vrf: true
+    missing_value_for_defaults: true
+    interface_enabled: true
```

Comment:
Do we need all these deviations for this test? Could we remove ones that are not necessary? (Same for the other vendors)

Reply:
it is needed.